BraintreeCore/src/main/java/com/braintreepayments/api/Configuration.kt

```diff
@@ -259,6 +259,12 @@ open class Configuration internal constructor(configurationString: String?) {
      */
     @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP) val venmoMerchantId: String
 
+    /**
+     * @return a boolean indicating whether ECD is enabled for this Venmo merchant.
+     * @suppress
+     */
+    @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP) val venmoEcdEnabled: Boolean
```

Comment:
Hey! What's ECD?

Reply:
Oh wait I see. Enriched Customer Data 👍

Reply:
```suggestion
    @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP) val venmoEnrichedCustomerDataEnabled: Boolean
```
nitpick: Since this is internal I wouldn't mind spelling this out fully. I think it'll help in the long run.

Reply:
That makes sense. I'll update it